### DISCUSSION TITLE
Welcome to bibmanager Discussions!

### DISCUSSION THREAD

**Discussion by pcubillos** [pcubillos/bibmanager#95] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AaronDavidSchneider** [comment]
Hey @pcubillos,

thanks a lot for this awesome tool. I have used it for quite some time now and it has really helped a lot to stay organised with my bib.

There is maybe only one small thing that could be helpful:
Labels.
My idea would be to have the possibility (optional) to add labels directly when you add an entry and are then able to filter entry's when you use bibm browse.

**Reply by pcubillos** [reply]
Hi Aaron,
Thanks for the support! Yes, I think it's a good idea too, and not a hard one to code.  The issue #75 also proposes a similar idea, but I haven't kept up much with this project lately due to work.  But seeing there's more interest in this idea I will take a look the next time I find some free time ;)

btw: If you have any specific vision of the functionality you would like to enable (what features/user interface), please feel free to point it out in #75

**Reply by AaronDavidSchneider** [reply]
Thanks for your response. I think I have overlooked #75. But just added my vision there :D
